test/src/main/java/org/apache/accumulo/test/metrics/MetricsIT.java

```diff
@@ -126,14 +150,37 @@ public void confirmMetricsPublished() throws Exception {
             } else if (flakyMetrics.contains(name)) {
               // ignore any flaky metric names seen
               // these aren't always expected, but we shouldn't be surprised if we see them
+            } else if (name.startsWith("accumulo.compactor.queue")) {
```

Comment:
It's out of scope of this PR, but we may want to add a defined metric prefix in the `MetricsProducer` for the compaction queue metrics 
e.g.: `METRICS_COMPACTOR_PREFIX + "queue"` 

This would allow us to directly link metric names to their uses in tests vs hardcoded references.

Reply:
I took care of this in 6a05e41.